test/examples-golden/positive/llvm/Const/Const.llvm

```diff
@@ -1,19 +1 @@
-; ModuleID = 'juvix-module'
-
-
- 
-
-
-%closure = type <{i8*, i8**}>
-
-
-declare external fastcc  i8* @malloc(i32)    
-
-
-declare external fastcc  void @free(i8*)    
-
-
-define external fastcc  i8 @main()    {
-main:
-  ret i8 10 
-}
```

Comment:
Seems like hte new lines in the llvm backend are removed and replaced with > which makes them harder to read

Reply:
> Seems like hte new lines in the llvm backend are removed and replaced with > which makes them harder to read

Yeah -- that's annoying, but without stripping the whitespace, I still hit the bug.

It is possible to see the real LLVM code, though, by using `compile` from the command line on the golden-test `.ju` file, so maybe this is tolerable?

Reply:
yeah as long as I can see the file unaltered as well I'm happy